RecoEgamma/EgammaElectronProducers/plugins/TrackingRegionsFromSuperClustersProducer.h

```diff
@@ -147,9 +154,16 @@ TrackingRegionsFromSuperClustersProducer(const edm::ParameterSet& cfg,
   deltaPhiRegion_        = regionPSet.getParameter<double>("deltaPhiRegion");
   deltaEtaRegion_        = regionPSet.getParameter<double>("deltaEtaRegion");
   useZInVertex_          = regionPSet.getParameter<bool>("useZInVertex");
+  useZInBeamspot_        = regionPSet.getParameter<bool>("useZInBeamspot");
+  useDefaultZ_           = regionPSet.getParameter<bool>("useDefaultZ");
```

Comment:
Agreed. I tried to think of a justification for my choice here and I found none. This is better